### DISCUSSION TITLE
Regarding Multi nodes and Multi GPU

### DISCUSSION THREAD

**Discussion by nimeeshrathi** [InteractiveComputerGraphics/SPlisHSPlasH#282] [General]
Could someone provide guidance on utilizing SPlisHSPlasH for multi-node or multi-GPU simulations?

**Comment by janbender** [comment]
If you build it using OpenMP (default), multi-core is automatically activated. Multi-GPU is not supported.

**Reply by nimeeshrathi** [reply]
Thanks for getting back  so quickly!
Is it possible that I can run the  simulation on multi-CPU ?

**Comment by janbender** [comment]
Yes, multi-CPU is supported.

**Reply by nimeeshrathi** [reply]
May I kindly request guidance on how to navigate the appropriate path towards doing so ?

**Reply by janbender** [reply]
All CPU cores are automatically used if OpenMP is actived in the CMake settings (which is the default setting).

**Reply by nimeeshrathi** [reply]
I apologize for any misunderstanding of the question. We currently have a cluster comprising 10 CPUs, and I am interested in executing the simulation across all 10 CPUs in order to reduce computational time.

**Reply by janbender** [reply]
I'm not sure if simple OpenMP supports your cluster configuration. We are using dual CPU systems. Probably you will need an MPI implementation, which is not provided by our library.
